DataStructures/Stacks/BalancedBrackets.java

```diff
@@ -15,71 +13,69 @@
  * returns true if S is nested and false otherwise.
  *
  * @author akshay sharma
- * @date: 2017-10-17
  * @author <a href="https://github.com/khalil2535">khalil2535<a>
- *
+ * @author shellhub
  */
 class BalancedBrackets {
 
     /**
+     * Check if {@code leftBracket} and {@code rightBrackets} is paired or not
+     *
+     * @param leftBracket   left bracket
+     * @param rightBrackets right bracket
+     * @return {@code true} if {@code leftBracket} and {@code rightBrackets} is paired,
+     * otherwise {@code false}
+     */
+    public static boolean isPaired(char leftBracket, char rightBrackets) {
```

Comment:
It should be `rightBracket`, not `rightBrackets`:

```java
/**
 * Check if {@code leftBracket} and {@code rightBracket} is paired or not
 *
 * @param leftBracket   left bracket
 * @param rightBracket right bracket
 * @return {@code true} if {@code leftBracket} and {@code rightBracket} is paired,
 * otherwise {@code false}
 */
public static boolean isPaired(char leftBracket, char rightBracket) {
    char[][] pairedBrackets = {
            {'(', ')'},
            {'[', ']'},
            {'{', '}'},
            {'<', '>'}
    };
    for (char[] pairedBracket : pairedBrackets) {
        if (pairedBracket[0] == leftBracket && pairedBracket[1] == rightBracket) {
            return true;
        }
    }
    return false;
}
```

Reply:
Updated! Please review again! @yanglbme